src/GNDStk/child.hpp

```diff
@@ -0,0 +1,545 @@
+   );
+}
+
+// child_t<void>--
+template<find FIND, class METADATA, class CHILDREN>
+inline auto operator--(
+   const child_t<void,FIND,detail::failure_t,METADATA,CHILDREN> &kwd, const int
+) {
+   return child_t<void,find::one,detail::failure_t,METADATA,CHILDREN>(
+      kwd.name, kwd.canBeTopLevel
+   );
+}
+
+// child_t<void>++
+template<find FIND, class METADATA, class CHILDREN>
+inline auto operator++(
+   const child_t<void,FIND,detail::failure_t,METADATA,CHILDREN> &kwd, const int
+) {
+   return child_t<void,find::all,detail::failure_t,METADATA,CHILDREN>(
+      kwd.name, kwd.canBeTopLevel
+   );
+}
+
+// child_t<TYPE>--
+template<class TYPE, find FIND, class CONVERTER, class METADATA, class CHILDREN>
+inline auto operator--(
+   const child_t<TYPE,FIND,CONVERTER,METADATA,CHILDREN> &kwd, const int
+) {
+   return child_t<TYPE,find::one,CONVERTER,METADATA,CHILDREN>(
+      kwd.name, kwd.converter, kwd.canBeTopLevel
+   );
+}
+
+// child_t<TYPE>++
+template<class TYPE, find FIND, class CONVERTER, class METADATA, class CHILDREN>
+inline auto operator++(
+   const child_t<TYPE,FIND,CONVERTER,METADATA,CHILDREN> &kwd, const int
+) {
+   return child_t<TYPE,find::all,CONVERTER,METADATA,CHILDREN>(
+      kwd.name, kwd.converter, kwd.canBeTopLevel
+   );
+}
+
+
+
+// -----------------------------------------------------------------------------
+// Macros
+// For child_t building. The macros don't handle the optional converter;
+// for that, just construct such an object directly.
+// -----------------------------------------------------------------------------
+
+// void (unspecified) result type (so, Node<>)
+#define GNDSTK_MAKE_CHILD_DEFAULT(name,FIND) \
+   inline const child_t<void,FIND> name(#name)
+
+// user-defined result type
+#define GNDSTK_MAKE_CHILD(result,name,FIND) \
+   inline const child_t<result,FIND> name(#name)
+
+// Note: we won't #undef these eventually, as we normally would,
+// because they're perfectly viable macros for users to invoke.
+
+
+
+// -----------------------------------------------------------------------------
+// Allowable top-level nodes
+// As given in LLNL-TR-774621-DRAFT
+// -----------------------------------------------------------------------------
+
+namespace child {
+
+// Note: the true values here mean "allowed as a top-level node"
+inline const child_t<void,find::one>
+   reactionSuite      ("reactionSuite",       true),
+   covarianceSuite    ("covarianceSuite",     true),
+   PoPs               ("PoPs",                true),
+   thermalScattering  ("thermalScattering",   true),
+   fissionFragmentData("fissionFragmentData", true);
+
+} // namespace child
+
+
+
+// -----------------------------------------------------------------------------
+// Keywords
+// Of type child_t
+//
+// fixme Eventually, many (or most (or all)) deserve a solid type,
+// not child_t's default.
+// -----------------------------------------------------------------------------
+
+namespace child {
```

Comment:
This appears again to be an old version of the code, specifically of src/GNDStk/child.hpp. Github's cluttered and convoluted interface strikes again. Now if I could just figure out what my comments are being "\centerline'd" instead of displayed like a normal person would want to see them.